django/db/backends/sqlite3/operations.py

```diff
@@ -70,21 +70,23 @@ def date_trunc_sql(self, lookup_type, field_name):
         # cause a collision with a field name).
         return "django_date_trunc('%s', %s)" % (lookup_type.lower(), field_name)
 
+    def _require_pytz(self):
+        if settings.USE_TZ and pytz is None:
+            raise ImproperlyConfigured("This query requires pytz, but it isn't installed.")
+
+    def datetime_cast_date_sql(self, field_name, tzname):
+        self._require_pytz()
+        return "django_datetime_date(%s, %%s)" % field_name, [tzname]
```

Comment:
Can you call this function `django_datetime_cast_date` for consistency? Everything else has cast_date / extract / trunc.